src/computeHashSum.in.lua

```diff
@@ -169,7 +169,7 @@ function main()
    end
    fh:write(s)
    if (HashSum:sub(1,1) == "@" ) then
-      HashSum = find_exec_path("sha1sum")
+      HashSum = find_exec_path("sha1sum") or find_exec_path("shasum")
```

Comment:
this is a fallback, in case Lmod hasn't been figured yet (hence the check for `@` above)

this is relied on to run the Lmod tests